modulefiles/miniconda3/4.5.12.lua

```diff
@@ -0,0 +1,27 @@
+help([[
+]])
+
+local pkgName = myModuleName()
+local pkgVersion = myModuleVersion()
+local shell=myShellType()
+
+conflict(pkgName)
+
+local base=pathJoin("/lustre/f2/pdata/esrl/gsd/contrib/miniconda3", pkgVersion)
```

Comment:
@climbfuji You had mentioned that things weren't working as expected on gaea. Were you able to install the RDHPCS-supported miniconda at version 4.5.12? If not, I think this file should reflect whatever version of miniconda you installed on gaea.

Reply:
pkgVersion is determined automatically from the module filename (a lua advantage), thus the version is documented in the name 4.5.12.lua

Reply:
I'm just curious if 4.5.12 is consistent with whatever version of miniconda has been installed on Gaea

Reply:
Got it. It's 4.8.3 actually. I bumped up the version in the PR wherever I could find it. Not sure that's what you wanted.